hack/bin/BUILD.bazel

```diff
@@ -58,6 +58,18 @@ genrule(
     visibility = ["//visibility:public"],
 )
 
+genrule(
+    name = "fetch_oc3",
+    srcs = select({
+        ":darwin": ["@oc_3_11_darwin//:file"],
```

Comment:
`oc cluster up` on a mac.. does this work?! 🤔

Reply:
It is a wrapper around the docker client, might work